### PR TITLE
BDCS-2628 Add auth hold codes section

### PR DESCRIPTION
**Pull request by sean-hill**
Make sure you've checked off all these things before submitting:

- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `dev` branch.
- [x] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

**Comment by sean-hill** [comment]
image

**Comment by sean-hill** [comment]
image

File: source/includes/api/auth/_hold-codes.md.erb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+### account_issue_account_locked
+
+Your account has been locked. Unlock your account directly through your provider and try again.
+
+<label class="tag">Biller Direct</label>
+
+### account_issue_verify_identity
+
+Please log in to your account to setup and verify your personal information.
```
Comment (kolinkodm) [inline source/includes/api/auth/_hold-codes.md.erb]: set up (two words)

File: source/includes/api/auth/_status.md.erb [hunk 2]
```diff
@@ -67,9 +67,9 @@ The auth's credentials are no longer valid and need to be reauthenticated using
 
 ![Bill Authenticate](<%= config.s3 %>/bill-authenticate.png)
 
-### hold 
+### hold
 
-For a number of reasons an auth can have the status of `hold`. When an auth is on hold we'll present you with a `reason` that you in turn can present to your users. To remove this status, the user must take the appropriate action as described in the `reason`. This usually involves them needing to login into their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services.
+For a number of reasons an auth can have the status of `hold`. When an auth is on hold we'll present you with a `reason` that you in turn can present to your users. To remove this status, the user must take the appropriate action as described in the `reason`. This usually involves them needing to login into their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services. In addition a [code](#auth-hold-codes) is provided for handling the hold programmatically if required.
```
Comment (kolinkodm) [inline source/includes/api/auth/_status.md.erb]: An auth's status can be `hold` for several reasons. When an auth is on hold, the response contains a `reason` for the hold that you can present to your users. To fix the hold, the user must perform the action described in the `reason`, which usually requires them to log in to their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services. 

Additionally, a [code](#auth-hold-codes) is provided for handling the hold programmatically, if required.

**Pull request merged**